packages/cli/src/controllers/activeWorkflows.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Service } from 'typedi';
+import { Authorized, Get, RestController } from '@/decorators';
+import { WorkflowRequest } from '@/requests';
```

Comment:
I'd've expected this to be linted:

```suggestion
import type { WorkflowRequest } from '@/requests';
```

Reply:
this is "required" for decorator metadata, even though this is just a type, and doesn't actually add anything in the compiled code.
in general, we should worry about this stuff in reviews anymore, and let the `consistent-type-imports` rule take care of it.

Reply:
> we should worry about this stuff in reviews anymore

Definitely, it just caught my eye and I got curious to know why this type was exempted :)